### PR TITLE
tweaked visual design for upgrades

### PR DESCRIPTION
**Pull request by paulirish**
before

image

after

image

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -320,4 +325,31 @@ font-family: 'Spectral SC', serif;
...
   .slider.round:before {
     border-radius: 50%;
-  }
\ No newline at end of file
+  }
+
+
+ /* monastery, blacksmith, univ images */
+ .div-upgrade-background-wrapper > .div-upgrade {
+    mix-blend-mode: multiply;
```
Comment (paulirish) [inline src/styles.css]: mix-blend-mode ftw. ;)

File: src/styles.css [hunk 2]
```diff
@@ -31,26 +31,30 @@ font-family: 'Spectral SC', serif;
 .div-upgrade-background-wrapper {
     margin: -1rem;
     display: inline-flex;
-    width: 45rem;
-    height: 10rem;
+    width: 720px;
+    height: 181px;
     background: url("https://treee.github.io/aoe-tech-tree-widget/build/images/chatbar_bg.png");
-    background-size: contain;
+    background-size: 720px 194px;
```
Comment (paulirish) [inline src/styles.css]: a hardcoded background-size stretches the image height a bit more.

File: src/styles.css [hunk 3]
```diff
@@ -31,26 +31,30 @@ font-family: 'Spectral SC', serif;
 .div-upgrade-background-wrapper {
     margin: -1rem;
     display: inline-flex;
-    width: 45rem;
-    height: 10rem;
+    width: 720px;
```
Comment (paulirish) [inline src/styles.css]: i switched some things from `rem` to `px` while i was working and forgot to change them back. sorry about that. I'm not attached to px.

File: src/aoe2/upgrade-changer.ts [hunk 4]
```diff
@@ -282,7 +263,7 @@ export class UpgradeChanger {
             template.addClass('disabled-upgrade');
         }
         const css: any = {
-            "background": `url("https://treee.github.io/aoe-tech-tree-widget/build/images/upgrade-icons/${upgrade}.png")`,
+            "background": `url("./images/upgrade-icons/${upgrade}.png")`,
```
Comment (paulirish) [inline src/aoe2/upgrade-changer.ts]: i believe this is safe everywhere and makes local dev a bit easier.

**Comment by paulirish** [comment]
oh, and apologies for not making it against your more recently updated ghpages. 

fwiw you can set up Pages to publish from master branch (right in settings), so you dont have to deal with the double. Or alternatively, you can use https://github.com/tschaub/gh-pages (manually, in a post-push hook, or a github action) to republish to ghpages branch.. and that way you could leave `./build` out of master.

File: src/aoe2/upgrade-changer.ts (hunk 4 above)
Comment (Treee) [reply src/aoe2/upgrade-changer.ts]: i like the improvement. smaller and easier to read

**Pull request merged**